lib/index.js

```diff
@@ -59,10 +59,12 @@ import {TextDocument} from 'vscode-languageserver-textdocument'
  */
 function vfileMessageToDiagnostic(message) {
   const diagnostic = Diagnostic.create(
-    message.position
-      ? fromPosition(message.position)
+    message.place
+      ? 'start' in message.place
+        ? fromPosition(message.place)
+        : {start: fromPoint(message.place), end: fromPoint(message.place)}
```

Comment:
It might be nice to add a function `fromPlace(place: Point | Position | undefined)` to `unist-util-lsp`.

Reply:
Should I interpret the :+1: as: _Go for it, implement it in `unist-util-lsp`?_